3DAmsterdam/Assets/Amsterdam3D/Scripts/Camera/CameraModeChanger.cs

```diff
@@ -77,6 +78,16 @@ public void MoveCameraToStreet(Transform cameraTransform, Vector3 position, Quat
             }
         }
 
+        public void ChangedPointFromUrl(string latLong)
+        {
+            string[] coordinates = latLong.Split(',');
+            var lat = double.Parse(coordinates[0]);
```

Comment:
Mogen wij dit afkorten? Zelf heb ik dat in mijn code niet gedaan, even dubbel checken.

Reply:
true, zal hem voluit schrijven